pkg/allocation/converters/converter_test.go

```diff
@@ -215,132 +215,8 @@ func TestConvertAllocationRequestToGameServerAllocation(t *testing.T) {
 			},
 		},
 		{
-			name:     "all fields are set",
```

Comment:
I'm sorry, I answered too quickly on this test. I think the nature of this test was to set all the built-in fields, but I think it was disabling all of the features to minimize the changes. Both this and the previous probably need to be restored.

Unfortunately, this part is going to require a bit deeper understanding of the test. You'll probably have to change this to just PlayerAllocationFilter=false and CountsAndList=false, then iterate on `want` until the expected output is correct. You can generally assume that the current code is correct and adjust the output to match.

Reply:
I apologize for choosing the issue myself. I have restored the code and I am not sure what changes are required to correct the output. Could you please help me?